configure

```diff
@@ -76,6 +76,7 @@ OPTIONS=(
   "libsystemd_daemon:no"
   "bintray_cache:yes"
   "ddci:yes"
+  "ddci_mcd:no"
```

Comment:
I don't think that we need two configuration options for ddci if we have the runtime config.

Reply:
I introduced the flag on the WEB UI later.
In fact this configuration option was meant to switch of the MCD code in case of problems after merging to master.
If you prefer to get rid of it, I will remove that.